### PR TITLE
Apply manifestowkr replicaset when use --replicaset label

### PR DESCRIPTION
**Pull request by qiujian16**
this generate a command to create manifestworkreplicaset

```
clusteradm create work -f xxx.yaml --placement default/placement --rs
```
will create manifestworkreplicaset in default ns and ref to placement

### CODE REVIEW

File: pkg/cmd/create/work/cmd.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 	o.ClusterOption.AddFlags(cmd.Flags())
 	cmd.Flags().StringVar(&o.Placement, "placement", "", "Specify an existing placement with format <namespace>/<name>")
 	cmd.Flags().BoolVar(&o.Overwrite, "overwrite", false, "Overwrite the existing work if it exists already")
+	cmd.Flags().BoolVarP(&o.UseReplicaSet, "replicaset", "rs", false, "Create ManifestworkReplicase instead of ManifestWork")
```
Comment (ycyaoxdu) [inline pkg/cmd/create/work/cmd.go:65]: `ManifestworkReplicase` -> `ManifestworkReplicaset`

Comment (serngawy) [inline pkg/cmd/create/work/cmd.go:65]: Better help message like "Create ManifestWorK for the associated placement's clusters using manifestWorkReplicaSet"

**Comment by ycyaoxdu** [comment]
@serngawy Hi, do you have any other comments?

**Review by serngawy — APPROVED** [review]

**Pull request merged**